safe-global/safe-wallet-web
# [Multichain] fix: Zksync network selector [SW-152]

## What it solves

Resolves SW-152

## How this PR fixes it

- Adds an `available` flag to each `ChainInfo` in the network selector
- Displays all networks but disables them depending on the flag
- Displays a message in case none of the networks are available

## How to test it

1. Open a Safe
2. Open the Network selector
3. Press Show all networks
4. Observe zkSync is visible but disabled
5. Switch to a zkSync safe
6. Open the network selector
7. Observe a message saying that this safe is not available on other networks

## Screenshots
<img width="422" alt="Screenshot 2024-09-16 at 16 21 19" src="https://github.com/user-attachments/assets/5d55b180-acf7-403e-a917-369aaec98500">
<img width="410" alt="Screenshot 2024-09-16 at 16 23 34" src="https://github.com/user-attachments/assets/ef810b08-cf81-44ad-aff2-0a77f29d6d50">

## Checklist
